README.md

```diff
@@ -379,6 +382,16 @@ Example: `is_quick_demo: True`
 - [future prospect] **Federated step-by-step increments**:
 
   See open [issue #105](https://github.com/SubstraFoundation/distributed-learning-contributivity/issues/105).
+  
+  Measures the performance increments on the global validation dataset after every minibatch training - Gives an estimation on how the model improved on every node
+  Method suited only for federated averaging (for now)
```

Comment:
I did not see any kind of safegard for that, it will be nice to have one. Like an assertion, or a test, to check if we really use fedavg, and raise an error if not.  (maybe in the contributivity file, when you check for the method to compute

Reply:
Adding an `assert` there would be a good idea indeed @arthurPignet . @jeromechambost does that seem OK?

Reply:
Indeed @arthurPignet 
Step by step method works and gives results for all learning approaches but is only relevant for federated averaging.
@bowni How about a test + warning rather than an assert so that the code can keep running when condition is not met ?

Reply:
Good idea!